packages/components/src/stepper/style.scss

```diff
@@ -0,0 +1,71 @@
+.woocommerce-stepper {
+	background: #fff;
+	box-shadow: 0 1px 3px rgba(0, 0, 0, 0.2),
+		0 2px 2px rgba(0, 0, 0, 0.12),
+		0 0 2px rgba(0, 0, 0, 0.14);
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	margin-bottom: $gap-large;
+	padding-left: $gap;
+	padding-right: $gap;
+
+	.woocommerce-stepper__step {
+		display: inline-flex;
+		align-items: center;
+		padding: $gap-small;
+		font-weight: 600;
+
+		svg {
+			display: none;
+		}
+
+		&.is-active,
+		&.is-complete {
+			color: #6802f3;
```

Comment:
The designs look like they introduce a new color scheme. Now (or in follow up) seems like a good time to introduce some variables.

Reply:
Great idea @psealock- added in https://github.com/woocommerce/woocommerce-admin/pull/2246/commits/1ea32e1013836d0a342c390400710b26ef7332f4